projects/rero/ng-core/src/lib/search-input/search-input.component.ts

```diff
@@ -43,7 +43,14 @@ export class SearchInputComponent {
     }
   }
 
+  /** Trim barcode if true */
+  @Input() trimBarcode = true;
```

Comment:
This is clearly not a barcode! You can use: `trimSearchText` or similar instead. Please update the comment and put more details such as remove trailing and leading spaces.

Reply:
Done